src/local-storage.js

```diff
@@ -0,0 +1,11 @@
+export const loadAuthToken= () => {
```

Comment:
Don't think we discuss local storage anywhere in web dev bootcamp at the moment, so that's something we'll definitely need to delve into.